### DISCUSSION TITLE
Perhaps increase --pageSidebarWidth when there is an active search

### DISCUSSION THREAD

**Discussion by dschissler** [pocketbase/pocketbase#2499] [Q&A]
I think that an easy improvement would be to have the collections search increase the width of the listing when a search is happening.  Already many of my collections are too long to fit.

**Comment by dschissler** [comment]
I also really wouldn't mind getting an extra 30 pixels by default on a wide screen.

**Reply by ganigeorgiev** [reply]
I don't think it will be a good idea to increase the sidebar width on search as it would cause too much "shifting" but I'll consider increasing the default width on wider screens.

_Side-note: The sidebar design most likely will change sometime in the future as several users complained about the lack of grouping/sorting but for now this has very low priority._

**Reply by dschissler** [reply]
Ok, so just a bit wider on wide screens, and also if you could give a hover over title should make things a lot better until a redesign.

**Reply by dschissler** [reply]
I'm prefixing my views with `v_` just because sometimes the view is what I'm using on the front end instead of a data collection.   Then I'm prefixing my admin views with `vadm` so that they stay together.  It also helps not to get confused and screwed up with filter permissions.  I can just know that `vadm` is wide open for group admin (not pb admin).
